Dockerfile

```diff
@@ -4,19 +4,27 @@ FROM debian:stable-slim AS mirrorbits
 ## and the risk of a breaking behavior is evaluated as low
 # hadolint ignore=DL3008
 RUN apt-get update && \
-  apt-get install --no-install-recommends -y tar curl ca-certificates && \
+  apt-get install --no-install-recommends -y tar curl ca-certificates git && \
   apt-get clean && \
   rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/*
 
+ARG go_version=1.21.3
+RUN mkdir -p /tmp/tools/ && \
+  curl -L "https://go.dev/dl/go${go_version}.linux-$(dpkg --print-architecture).tar.gz" --output /tmp/tools/go.tar.gz && \
+  tar -C /tmp/tools/ -xzf /tmp/tools/go.tar.gz
+
 ARG mirrorbits_version=v0.5.1
-RUN mkdir /mirrorbits && \
-  curl -L https://github.com/etix/mirrorbits/releases/download/${mirrorbits_version}/mirrorbits-${mirrorbits_version}.tar.gz -O && \
-  tar xvzf /mirrorbits-${mirrorbits_version}.tar.gz -C /
+ARG mirrorbits_current_commit=9189dc7
+# hadolint ignore=DL3003
+RUN git clone https://github.com/etix/mirrorbits /tmp/tools/mirrorbits && \
+  cd /tmp/tools/mirrorbits && \
+  git checkout "${mirrorbits_current_commit}" && \
```

Comment:
I've tried `git checkout "${mirrorbits_version}"` but the repo content is missing vendors and other build artifacts at this tag.

Switching to the current commit and keeping it fixed is the workaround I came up with to avoid building on new and unknown commits that could be added to https://github.com/etix/mirrorbits.

Reply:
How does it fare if:

- Keep using the 0.5.1 tag
- Build with the same instructions as their travis yaml file: https://github.com/etix/mirrorbits/blob/v0.5.1/.travis.yml (includes the `before_install to get the exact protobuf version)